app/src/main/res/values-fa/strings.xml

```diff
@@ -1,25 +1,123 @@
-    <string name="whats_supported">چه چیز پشتیبانی میکند؟</string>
-    <string name="style">طرح</string>
-    <string name="device_setting">تنظیمات دستگاه</string>
+    <string name="cancel">رد کردن</string>
+    <string name="unknown_filename">نام پرونده ناشناس</string>
+    <string name="whats_supported">چه چیزی پشتیبانی می‌شود؟</string>
+    <string name="style">سبک</string>
+    <string name="device_setting">تنظیم دستگاه</string>
     <string name="light">روشن</string>
-    <string name="dark">تاریک</string>
-    <string name="continue_anyway">ادامه دادن به هرطریق</string>
+    <string name="dark">تیره</string>
+    <string name="continue_anyway">به هر حال ادامه دهید</string>
     <string name="notification_write_progress_content">%1$s به %2$s</string>
-    <string name="permission_denied_for_usb_device">دسترسی به فلش %1$s امکان پذیر نیست</string>
-    <string name="select_a_usb_drive">انتخاب یک فلش</string>
-    <string name="notif_channel_job_progress_name">پیشرفت کارایش</string>
-    <string name="notification_write_progress_title">درحال نوشتن فایل..</string>
-    <string name="notification_verify_progress_title">درحال اعتبارسازی فایل…</string>
-    <string name="notification_verify_no_notifications_text">فعال کردن اعلان‌ها برای پیشرفت • تأیید %1$s در برابر %2$s</string>
-    <string name="write_finished">با موفقیت رایت شد.</string>
-    <string name="success_notif_content_text">%1$s با موفقیت در %2$s رایت شد</string>
-    <string name="write_failed">نوشتن ناموفق بود</string>
-    <string name="notif_channel_job_progress_desc" comment="Notification channel description">برای نشان دادن درصد عملیات کار فعلی استفاده می شود</string>
-    <string name="action_required">نیازمند به دسترسی</string>
-    <string name="notif_channel_job_result_name">نتیجه عملیات</string>
+    <string name="permission_denied_for_usb_device">دسترسی برای دستگاه USB %1$s رد شد</string>
+    <string name="select_a_usb_drive">یک USB USB را انتخاب کنید</string>
+    <string name="notif_channel_job_progress_name">پیشرفت کار</string>
+    <string name="notification_write_progress_title">تصویر در حال نوشتن است…</string>
+    <string name="notification_verify_progress_title">تصویر در حال پذیرش است…</string>
+    <string name="notification_verify_no_notifications_text">آگاه‌ساز‌ها را برای مشاهده پیشرفت فعال کنید • پذیرش %1$s در مقابل %2$s</string>
+    <string name="write_finished">نوشتن پایان یافت</string>
+    <string name="success_notif_content_text">%1$s به درستی به %2$s نوشته شد</string>
+    <string name="write_failed">نادرستی در نوشتن</string>
+    <string name="notif_channel_job_progress_desc" comment="Notification channel description">برای نشان دادن پیشرفت کار کنونی بکارگرفته می‌شود</string>
+    <string name="action_required">کنش مورد نیاز</string>
+    <string name="notif_channel_job_result_name">پایان کار</string>
+    <string name="looking_for_usb_drives">در حال جستجوی USBهای USB</string>
+    <string name="usb_drive">USB USB</string>
```

Comment:
Hey @alr86, thanks for your submission, I really appreciate it!

Since its public release I've been double-checking translations with ChatGPT to make sure trolls don't add slurs and fend off unpleasant surprises.

I submitted yours and it reported back the following:

> ### Areas for Improvement
> 
> - Repetitive Use of "USB":
>   In some places, "USB USB" is repeated, such as:
>	```xml
>   <string name="usb_drive">USB USB</string>
>   <string name="the_usb_drive_was_unplugged">USB USB جدا شد</string>
>	```
>   This should be corrected to avoid redundancy. For example, "درایو USB" (USB drive) could replace "USB USB."
> - Literal Translations:
>   Some translations are very literal, which might affect readability or naturalness in Farsi. For example: "یاد بگیرید که به چیست" for "Learn what it means" could be improved to "معنی آن را یاد بگیرید" for better clarity.

Please bear with me since I know nothing about Persian and it's well known that ChatGPT often spits out a lot of false information, but the suggestions seemed plausible to me.

Since you used "USB USB" in many places I suppose it's a common expression, but I wanted to double-check with you before merging this update.

Thanks again for your help, it's really appreciated, and have a great day!

Reply:
Thank you very much and thanks for fixing the issues!

I'm going to close this pull request since Weblate seems to have gotten stuck. A new one should be created automatically I hope.

Reply:
Ok, your changes went in with #245. Thanks again!